concordia/settings_dev.py

```diff
@@ -19,7 +19,7 @@
 CELERY_BROKER_URL = "pyamqp://guest@localhost"
 CELERY_RESULT_BACKEND = "rpc://"
 
-S3_BUCKET_NAME = "concordia-staticpages"
+S3_BUCKET_NAME = "crowd-dev-content"
```

Comment:
I'm a bit tired of having to manage these around the settings files. What do you think about changing `settings_template.py` to default to `S3_BUCKET_NAME = os.getenv("S3_BUCKET_NAME", f"crowd-{CONCORDIA_ENVIRONMENT}-content")` (after the environment name is pulled in, of course)?

I'm also wondering whether this is a good time to setup more than one storage backend with the goal towards eventually having bags, etc. available as public downloads which might not go through Django.

Reply:
Okay, I'm game to set up a separate S3 bucket for the exported content. I think it's also a good idea to figure out the S3 bucket names based on the environment. However the buckets are named dev test stage and prod, so it'd have to be based on ENV_NAME instead of CONCORDIA_ENVIRONMENT

Reply:
On second look, I don't think we need to change this particular setting. This is only for local runserver development using `settings_dev` where presumably no dedicated storage backend has been set up. In docker deployments, the S3 bucket name is passed as an env var to the docker container. I'll set up a separate bucket for exported content, but I think we should just stick with passing both the bucket names as env vars to docker like we do for other deployment-specific settings. If we want to programmatically generate the bucket names, the place to do that would be in the cloud formation template.

Reply:
> where presumably no dedicated storage backend has been set up

Does that mean this should just be blank then so the `if s3_bucket` check will fall back to local storage?

Reply:
Yes, let's make it blank on the checked-in version.